scripts/python-skywater-pdk/skywater_pdk/generate-resources-rst.py

```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/env python3
```

Comment:
I'm wondering if this should just be part of the sphinx documentation?

When does this tool run? Should it be run as part of building the documentation or as a scheduled github action (with the results committed to the repository)?

Reply:
my idea was that this script should be executed like separately like rest of the generating scripts.